samples/ftanomaly/TAnomalyTask.cpp

```diff
@@ -76,30 +77,36 @@ bool AnomalyTask::ParseBody() {
         if (!document.IsObject()) {
             DARWIN_LOG_ERROR("AnomalyTask:: ParseBody:: You must provide a valid Json");
             return false;
+        }else{
+            DARWIN_LOG_DEBUG("AnomalyTask:: ParseBody:: Body is a json");
         }
 
-        //********LEARNING MODE*********//
-        if (document.HasMember("learning_mode")){
-            DARWIN_LOG_DEBUG("AnomalyTask:: ParseBody:: Body has learning mode");
-            if (!document["learning_mode"].IsString()) {
-                DARWIN_LOG_CRITICAL("AnomalyTask:: ParseBody:: \"learning_mode\" needs to be a string");
+        //********DETECTION MODE*********//
+        if (document.HasMember("detection_mode")){
+            DARWIN_LOG_DEBUG("AnomalyTask:: ParseBody:: Body has detection mode");
+            if (!document["detection_mode"].IsString()) {
+                DARWIN_LOG_CRITICAL("AnomalyTask:: ParseBody:: \"detection_mode\" needs to be a string");
                 return false;
             }
 
-            std::string learning_mode = document["learning_mode"].GetString();
+            std::string detection_mode = document["detection_mode"].GetString();
 
-            if (learning_mode=="on"){
-                _learning_mode = true;
-            } else if (learning_mode=="off"){
-                _learning_mode = false;
+            if (detection_mode=="on"){
+                _detection_mode = true;
+            } else if (detection_mode=="off"){
+                _detection_mode = false;
             } else {
-                DARWIN_LOG_CRITICAL("AnomalyTask:: ParseBody:: \"learning_mode\" has to be either \"on\" or \"off\"");
+                DARWIN_LOG_CRITICAL("AnomalyTask:: ParseBody:: \"detection_mode\" has to be either \"on\" or \"off\"");
```

Comment:
```suggestion
                DARWIN_LOG_ERROR("AnomalyTask:: ParseBody:: \"detection_mode\" has to be either \"on\" or \"off\"");
```
This logger does not need to be `critical` as this error does not prevent the filter from running.

Reply:
Why was this resolved without any change or comment ?